04/app.js

```diff
@@ -1,9 +1,27 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: 'Adam',
+	lastName: 'Nowak',
+	born: {
+		day: '28',
+		month: '07',
+		year: '1985',
+	},
+};
+
+function brithdayChecker(name) {
+	const userDay = this.born.day;
+	const userMonth = this.born.month;
+
+	const dateDay = new Date().getDate();
+	const dateMonth = new Date().getMonth() + 1;
+
+	if (dateMonth === 12) {
+		dateMonth = 0;
+	}
```

Comment:
Chyba to tak nie działa :) - styczeń to 0, a grudzień 11. W Twoim rozwiązaniu grudzień to 0 :P